compiler/dyno/include/chpl/parsing/parsing-queries.h

```diff
@@ -147,6 +148,18 @@ uast::Function::ReturnIntent idToFnReturnIntent(Context* context, ID id);
  */
 bool functionWithIdHasWhere(Context* context, ID id);
 
+void setConfigParams(Context* context, ConfigParamList keys);
+
+const
+ConfigParamList& configParams(Context* context);
```

Comment:
I think we can come up with a better name for this and related functions (and `ConfigParamList`). The issue is that `-sbla=bla` on the command line can also change the default value of a `config var` or `config const`. So it is not strictly about `config param` at all.

Perhaps `compileConfigSettings` or something along those lines would be better?